xknx/remote_value/remote_value_step.py

```diff
@@ -40,19 +40,23 @@ def payload_valid(self, payload):
         """Test if telegram payload may be parsed."""
         return isinstance(payload, DPTBinary)
 
+    # from KNX Association System Specifications AS v1.5.00:
+    # 1.007 DPT_Step   0 = Decrease 1 = Increase
+    # 1.008 DPT_UpDown 0 = Up       1 = Down
```

Comment:
1.008 has its own RemoteValue found in remote_value_updown.py .
It would be nice if we combine them to a single file with these two classes - they are essentially the same and could easily inherit from a RemoteValueBinary base class or just from RemoteValueSwitch().